src/test/java/com/faforever/api/clan/ClanControllerIntegrationTest.java

```diff
@@ -35,44 +29,25 @@
 import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
 import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
 
+
 @RunWith(SpringRunner.class)
 @SpringBootTest
-@Ignore("This needs to be cleaned up big time.")
+@Import(TestDatabase.class)
+@Transactional
 public class ClanControllerIntegrationTest {
```

Comment:
All integration tests will share a common test setup (including `@Transactional`) which is why there should be an `AbstractIntegrationTest` class.